images/release/golang-1.8/Dockerfile.altarch

```diff
@@ -6,23 +6,18 @@
 #
 FROM openshift/origin-base
 
-ENV VERSION=1.6.3 \
+ENV VERSION=1.8.3 \
     GOARM=5 \
     GOPATH=/go \
-    GOROOT=/usr/local/go \
     OS_VERSION_FILE=/go/src/github.com/openshift/origin/os-version-defs \
     TMPDIR=/openshifttmp
-ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+ENV PATH=$PATH:$GOPATH/bin
 
-ADD *.rpm /tmp/origin-rpm/
 RUN mkdir $TMPDIR && \
-    INSTALL_PKGS="make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel" && \
+    INSTALL_PKGS="make gcc zip mercurial krb5-devel bsdtar bc rsync bind-utils file jq tito createrepo openssl gpgme gpgme-devel libassuan libassuan-devel protobuf-compiler golang" && \
     yum install -y $INSTALL_PKGS /tmp/origin-rpm/*.rpm && \
     rpm -V $INSTALL_PKGS && \
     yum clean all && \
-    curl -L https://github.com/google/protobuf/releases/download/v3.0.0-beta-4/protoc-3.0.0-beta-4-linux-x86_64.zip | bsdtar -C /usr/local -xf - && \
-    chmod uga+x /usr/local/bin/protoc && \
```

Comment:
It's comparing a new Dockerfile for altarch against one from the removed golang-1.6 directory. protobuf-compiler and golang are installed from RPM. The main driver for switching to RPM was that there are no pre-compiled binaries for protoc for ppc64le, aarch64, or s390x.

Similarly, golang doesn't provide pre-compiled binaries for aarch64 or s390x.